module/planning/FallingRelaxPlanner/src/FallingRelaxPlanner.cpp

```diff
@@ -93,21 +94,22 @@ namespace module::planning {
                 // We are falling
                 if (falling) {
                     // We are falling! Relax the limbs!
-                    log<NUClear::INFO>("Falling:",
-                                       "Gyroscope Magnitude:",
-                                       gyro_mag_state == State::FALLING    ? "FALLING"
-                                       : gyro_mag_state == State::UNSTABLE ? "UNSTABLE"
+                    log<NUClear::TRACE>("Falling:",
```

Comment:
I think this is more of a debug line. Keep it as debug and change the config to be info or higher by default. Debug should be log lines that help you debug behaviour. Info lines should be for people. Trace should be for debugging code

Reply:
you mean should be changed to debug, they were both originally info

Reply:
Yes that